transport/src/main/java/io/camunda/zeebe/transport/stream/impl/ClientStreamManager.java

```diff
@@ -73,7 +75,7 @@ void remove(final ClientStreamId streamId) {
     final var serverStream = registry.removeClient(streamId);
     serverStream.ifPresent(
         stream -> {
-          LOG.debug("Removing aggregated stream [{}]", stream.getStreamId());
+          LOG.debug("Removing aggregated stream [{}]", stream.streamId());
```

Comment:
Good point. These aren't really on the hot path, since they're all about adding/removing streams, which shouldn't happen very often. Errors will lead to retries, but these are all with back off. That said, a bug could possibly cause these to be very verbose as well. I've found the debug log useful as we were developing the feature, but maybe once it's stable it's not very useful anymore :thinking: